src/components/KTabs/KTabs.vue

```diff
@@ -98,57 +99,69 @@ watch(() => props.modelValue, (newTabHash) => {
+    margin-bottom: var(--kui-space-70, $kui-space-70);
+    padding: var(--kui-space-0, $kui-space-0) var(--kui-space-70, $kui-space-70);
 
     .tab-item {
       cursor: pointer;
+      /* stylelint-disable-next-line @kong/design-tokens/use-proper-token */
+      margin-bottom: calc(-1 * var(--kui-border-width-10, $kui-border-width-10)); // to make the active item border appear on top of the bottom border
+      padding-bottom: var(--kui-space-40, $kui-space-40);
       position: relative;
+      transition: border-bottom $kongponentsKongIconSelector;
 
       .tab-link {
-        font-size: inherit;
+        border-radius: var(--kui-border-radius-30, $kui-border-radius-30);
+        color: var(--kui-color-text-neutral, $kui-color-text-neutral);
+        display: inline-flex;
+        gap: var(--kui-space-40, $kui-space-40);
+        padding: var(--kui-space-30, $kui-space-30) var(--kui-space-50, $kui-space-50);
         text-decoration: none;
+        transition: color $kongponentsTransitionDurTimingFunc, background-color $kongponentsTransitionDurTimingFunc, box-shadow $kongponentsTransitionDurTimingFunc;
 
-        a,
-        :deep(a) {
+        a, :deep(a) {
+          color: var(--kui-color-text, $kui-color-text);
           text-decoration: none;
         }
-      }
 
-      .tab-link,
-      .tab-link:not(.has-panels) :deep(> *) {
-        color: var(--kui-color-text, $kui-color-text);
+        &:focus-visible {
+          background-color: var(--kui-color-background-neutral-weaker, $kui-color-background-neutral-weaker);
+          box-shadow: var(--kui-shadow-focus, $kui-shadow-focus);
+          color: var(--kui-color-text, $kui-color-text);
+          outline: none;
+        }
       }
 
-      .tab-link.has-panels,
-      .tab-link:not(.has-panels) :deep(> *) {
-        display: inline-block;
-        padding: var(--kui-space-60, $kui-space-60);
+      &:hover:not(.disabled) {
+        .tab-link {
+          background-color: var(--kui-color-background-neutral-weaker, $kui-color-background-neutral-weaker);
+        }
       }
 
-      &:not(:first-of-type) {
-        margin-left: var(--kui-space-40, $kui-space-40);
-      }
+      &.active {
+        border-bottom: var(--kui-border-width-20, $kui-border-width-20) solid var(--kui-color-border-decorative-purple, $kui-color-border-decorative-purple);
 
-      &:not(:last-of-type) {
-        margin-right: var(--kui-space-40, $kui-space-40);
+        .tab-link {
+          color: var(--kui-color-text, $kui-color-text);
+        }
       }
 
-      &.active,
-      &:hover {
-        .tab-link.has-panels,
-        .tab-link:not(.has-panels) :deep(> *) {
-          border-bottom: var(--kui-border-width-30, $kui-border-width-30) solid $tmp-color-aqua-50; // teal-300 is now aqua-50
-          color: var(--kui-color-text, $kui-color-text);
+      &.disabled {
+        cursor: not-allowed;
+
+        .tab-link {
+          color: var(--kui-color-text-disabled, $kui-color-text-disabled);
```

Comment:
![image](https://github.com/Kong/kongponents/assets/2229946/a3506208-b65b-4c12-9092-1e542df0d856)
The disabled color and the inactive color are almost indistinguishable.

Reply:
Yes. No big deal I think since we won't often have disabled tabs but something we should address with design for sure.